core/templates/services/rte-helper-modal.controller.spec.ts

```diff
@@ -356,21 +356,22 @@ describe('RteHelperModalComponent', () => {
       );
       expect(component.isErrorMessageNonempty()).toBe(true);
       expect(component.errorMessage).toBe(
-        'Please ensure that each title and content are filled'
+        'Please ensure that tab 1 title is filled.'
       );
       flush();
     }));
 
     it('should disable save button and display error message', fakeAsync(() => {
       component.ngOnInit();
       flush();
+      component.customizationArgsForm.value[0][0].title = 'Tab 1';
       component.customizationArgsForm.value[0][1].content = '';
       component.onCustomizationArgsFormChange(
         component.customizationArgsForm.value
       );
       expect(component.isErrorMessageNonempty()).toBe(true);
       expect(component.errorMessage).toBe(
-        'Please ensure that each title and content are filled'
+        'Please ensure that tab 2 content is filled.'
```

Comment:
Please ensure that the content of tab 2 is filled

Reply:
Done!